sdk/eventgrid/azure-eventgrid/tests/recordings/test_eg_publisher_client.test_send_cloud_event_data_as_list.yaml

```diff
@@ -15,7 +15,7 @@ interactions:
       Content-Type:
       - application/cloudevents-batch+json; charset=utf-8
       User-Agent:
-      - azsdk-python-eventgridpublisherclient/unknown Python/3.7.3 (Windows-10-10.0.18362-SP0)
+      - azsdk-python-core/1.8.2 Python/3.7.3 (Windows-10-10.0.18362-SP0)
```

Comment:
When you redefine your own pipeline you need to be sure you configure the policies correctly for the context of this package. Here you're missing the sdk_moniker for the UserAgent policy